docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3"
+
+services: 
+     
+  db:
+    image: mariadb:latest
+    container_name: mariadb-server
```

Comment:
Do not use container names. It might cause confusion if there are multiple compose projects running.